### PR TITLE
[sync] 2020/8/31

### PR DESCRIPTION
**Pull request by chenglu**
ref #374

### CODE REVIEW

File: src/docs/get-started/flutter-for/web-devs.md [hunk 1]
```diff
@@ -119,10 +119,18 @@ the top left, by default.
...
 
 In Flutter, you set the background color using a
 [`Container`][]’s `decoration` property.
+=======
+In Flutter, you set the background color using the 
+`color` property or the `decoration` property of a [`Container`][]. However, you cannot
+supply both, since it would potentially result in the decoration drawing over
+the background color. The `color` property should be preferred when the background is a simple color.
+For other cases, such as gradients or images, use the `decoration` property.
+>>>>>>> 2341c639fdfcf7aeef464a95dabab604bbf12532
```
Comment (AlexV525) [inline src/docs/get-started/flutter-for/web-devs.md:133]: Here we have some unfinished merge process.

File: src/docs/development/platform-integration/c-interop.md [hunk 2]
```diff
@@ -338,6 +359,7 @@ in binary form, use the following instructions:
    downloaded from a repository, such as
    JCenter.
 
+
```
Comment (AlexV525) [inline src/docs/development/platform-integration/c-interop.md:362]: Extra line?

**Review by AlexV525 — COMMENTED** [review]
Two tiny problems. 🤣

File: src/docs/development/platform-integration/c-interop.md (hunk 2 above)
Comment (chenglu) [reply src/docs/development/platform-integration/c-interop.md]: fixed

File: src/docs/get-started/flutter-for/web-devs.md (hunk 1 above)
Comment (chenglu) [reply src/docs/get-started/flutter-for/web-devs.md]: fixed.

**Review by AlexV525 — APPROVED** [review]
image

**Pull request merged**